Emby.Server.Implementations/Library/MediaSourceManager.cs

```diff
@@ -388,6 +388,18 @@ private void SetDefaultSubtitleStreamIndex(MediaSourceInfo source, UserItemData
             }
 
             var preferredSubs = NormalizeLanguage(user.SubtitleLanguagePreference);
+            var preferredCodec = user.SubtitleCodecPreference;
+            var preferredCodecs = preferredCodec switch
+            {
```

Comment:
Maybe add an option for PGS (HDMV PGS) as well? Quite a common format.

Reply:
Good point. None of the image based formats are handled. Jellyfin only had constants defined for text based formats. I'll look over the codebase and check if there's any other places image based subs are mentioned.